### DISCUSSION TITLE
Does the mod working?

### DISCUSSION THREAD

**Discussion by yaqot15** [onimp/oni_multiplayer#166] [Q&A] [answered]
Hi i really apricate your hard work guys for this mod, But does it still working ? 
because i have try to install it with my friend we haven't figure it out how to ….. Please help me if you could

**Comment by zuev93** [comment, marked as answer]
Hi @yaqot15 yes it works.
What issues do you face?
Explain more otherwise we won't be able to help you.
Also as a default and most often case: please take a look into #114

**Reply by yaqot15** [reply]
Well our problem is we have downloaded the mod but we didn't see any multiplayer UI even we did all steps for how to install the mod ... so idk maybe we have done something wrong

**Reply by yaqot15** [reply]
Ok i fixed it Thanks ... seems like i have to download first any mod from Steam then it will pop out the Multiplayer mod <3
